packages/cli/src/zosfiles/download/Download.options.ts

```diff
@@ -16,7 +16,8 @@ import i18nTypings from "../-strings-/en";
 // Does not use the import in anticipation of some internationalization work to be done later.
 const strings = (require("../-strings-/en").default as typeof i18nTypings).DOWNLOAD.OPTIONS;
 
-const maxConcurrentRequestsMaxValue = 99999;
+/** @internal */
+export const maxConcurrentRequestsMaxValue = 99999;
```

Comment:
Do we want to decrease this number for a future PR? If so, any suggestions on what the limit should be?

Reply:
maybe 100 (if it's too low, then 1000) 😋

Reply:
100 seems reasonable. I tried 1000 and would not recommend it.